### DISCUSSION TITLE
Failed to construct 'CustomElement' solution + 1 size q

### DISCUSSION THREAD

**Discussion by MrDrone-ru** [material-components/material-web#5496] [Q&A] [answered]
Hi everyone!
I used a quickstart guide with bundle building and encountered some problems, one of which is solved quite simply but very non-obvious especially for a js newbie. So I'll post the solution here, maybe someone will find it useful.
So, I plugged this lib into `<head>`, then added a button - everything works. After that I added `<md-outlined-select>` with one option with attribute` selected`, but instead of selected option I saw only empty field. Next I tried to add a component `<md-tabs>` and got an error `Failed to construct 'CustomElement': The result must not have attributes` instead of my tabs.
Apparently material lib needs to see its components in the DOM tree to construct them properly. 
So the solution is to **move the script** to the end of page or **add an `defer` attribute** to it. 

But I have another question and it's unanswered: how to change the text size in the dropdown menu of an `<md-outlined-select>`? I found css selector `md-outlined-select::part(menu)` and tried to change different properties (from md-comp-menu-item tokens) but it didn't work.

**Comment by asyncliz** [comment, marked as answer]
I believe the tabs attribute hydration error was fixed in https://github.com/material-components/material-web/pull/5463 and released in [`v1.3.0`](https://github.com/material-components/material-web/releases/tag/v1.3.0). Can you try updating to see if that fixes it?

For styling, you should be able to set `--md-menu-item-*` custom properties on the `<md-select-option>`.

```html
<style>
  md-select-option {
    --md-menu-item-label-text-size: 24px;
    --md-menu-item-label-text-line-height: 32px;
  }
</style>
<md-outlined-select label="Select">
  <md-select-option>First</md-select-option>
  <md-select-option>Second</md-select-option>
</md-outlined-select>
```

[Playground demo](https://lit.dev/playground/#gist=3b65cb152be57c4f1310be32cc539f39)

**Reply by MrDrone-ru** [reply]
Big THANKS))
I thought the `npm install` command installs the latest release version, but for me it doesn't. It fixed the tabs, but it didn't fix the selected option display, `defer` still more reliable. But construction without errors - it's good.
